code/game/turfs/simulated/floor/plating.dm

```diff
@@ -269,6 +269,12 @@
 					var/mob/living/live = L.buckled
 					if("lava" in live.weather_immunities)
 						continue
+			if(ishuman(L))
```

Comment:
I'll merge this tonight if you want to remove the lava upgrade and figure it out in another PR